src/seedu/addressbook/AddressBook.java

```diff
@@ -484,15 +500,26 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      */
     private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
+        Collection<String> lowerCaseKeywords = convertPersonsSearchKeywordsToLowerCase(keywords);
         for (String[] person : getAllPersonsInAddressBook()) {
-            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person).toLowerCase()));
+            if (!Collections.disjoint(wordsInName, lowerCaseKeywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
```

Comment:
When you modify the behavior, remember to update documentation and tests as well. The comments should be updated too.